docs/source/dataset_script.mdx

```diff
@@ -270,85 +270,27 @@ def _generate_examples(self, filepath):
 ## Testing data and checksum metadata
 
 We strongly recommend adding testing data and checksum metadata to your dataset to verify and test its behavior. This ensures the generated dataset matches your expectations.
-Testing data and checksum metadata are mandatory for datasets stored in the GitHub repository of the 🤗 Datasets library.
+Testing data and checksum metadata are mandatory for datasets stored in the 🤗 Datasets GitHub repository.
 
 <Tip warning={true}>
 
-Make sure you run all of the following commands **from the root** of your local `datasets` repository.
+Make sure you run the following command **from the root** of your local `datasets` repository.
```

Comment:
This was necessary for pytest and the dummy data, but not for the datasets-cli test command. I think you can remove it :)
```suggestion
```